aries-site/src/pages/components/radiobuttongroup.mdx

```diff
@@ -19,7 +19,7 @@ import {
 
 ### Best Practices
 
-- Keep labels concise and simple. Avoid lengthy descriptions, however err on clarity over brevity.
+- Keep labels concise and simple. Avoid lengthy descriptions, however error on clarity over brevity.
```

Comment:
Err is a verb (i.e. to make a mistake) and is used correctly here. Error is a noun and is incorrect here.

```suggestion
- Keep labels concise and simple. Avoid lengthy descriptions, however err on clarity over brevity.
```

Reply:
https://www.merriam-webster.com/dictionary/err

https://www.beedictionary.com/common-errors/error_vs_err

Reply:
thanks for this I never knew that! I also thought it was a typo

Reply:
Thanks for the clarification @halocline Learned something new